src/model.c

```diff
@@ -255,6 +287,115 @@ static bool parse_int(char **buffer, int *i)
     return true;
 }
 
+struct model *model_load_from_ply(const char *fname)
+{
+    FILE *fp = fopen(fname, "r");
+    if (!fp)
+    {
+        fprintf(stderr, "ERROR: failed to load file \"%s\".\n", fname);
+        return NULL;
+    }
+
+    // Create a new model
+    struct model *model = model_init();
+
+    // Read each line of the file
+    char buffer[256];
+
+    int vertex_count = 0;
+    int face_count = 0;
+
```

Comment:
My understanding is that all PLY files start with a `ply` line. I think we should check for that and throw an exit(1) printing "invalid format" to stderr if it is missing.

Also, I read that there are both a binary PLY  format and a text PLY format. How can we check that this is indeed a text PLY file?

Reply:
It is still pending, but we should check for a line with
```
format ascii 1.0
```
in the header.